homeassistant/components/netatmo/climate.py

```diff
@@ -82,90 +143,274 @@ def temperature_unit(self):
     @property
     def current_temperature(self):
         """Return the current temperature."""
-        return self._data.current_temperature
+        return self._data.room_status[self._room_id]['current_temperature']
```

Comment:
Can you confirm that there is no IO happening here?

Reply:
No the I/O is only in: setup_platform, HomeData::setup, ThermostatData::setup, and ThermostatData::update. (the pyatmo stuff)